webpack/ForemanInventoryUpload/Components/AccountList/AccountListActions.js

```diff
@@ -58,7 +59,15 @@ export const restartProcess = (accountID, activeTab) => dispatch => {
     processStatusName = 'generate_report_status';
   }
 
-  API.post(inventoryUrl(`${accountID}/${processController}`));
+  API.post(inventoryUrl(`${accountID}/${processController}`)).catch(error => {
+    dispatch(
+      addToast({
+        sticky: true,
+        type: 'error',
+        message: error.message,
+      })
+    );
+  });
```

Comment:
I think it is more readable inside a `try -> catch` block, 
and it will also prevent the dispatch of `INVENTORY_PROCESS_RESTART` if not needed, e,g:

```suggestion
   try {
    await API.post(inventoryUrl(`${accountID}/${processController}`))
    dispatch({
      type: INVENTORY_PROCESS_RESTART,
      payload: {
        accountID,
        processStatusName,
      },
    });
  } catch(error){
    dispatch(
      addToast({
        sticky: true,
        type: 'error',
        message: error.message,
      })
    );
  }; 
};
```

Reply:
Since the method is not async, I cannot use `await`. If the dispatch of the restart is needed only on success, I can change the code to 
```suggestion
  API.post(inventoryUrl(`${accountID}/${processController}`)).then(
  () => {
    dispatch({
      type: INVENTORY_PROCESS_RESTART,
      payload: {
        accountID,
        processStatusName,
      },
    });
  }, 
  error => {
    dispatch(
      addToast({
        sticky: true,
        type: 'error',
        message: error.message,
      })
    );
  });
```

Reply:
> Since the method is not async, I cannot use `await`

you can add `async` just before passing the `dispatch` here: https://github.com/theforeman/foreman_rh_cloud/pull/351/files#diff-cc6147ee1ab873c90de32fe3b134f1aeR50

e.g:
`export const restartProcess = (accountID, activeTab) => async dispatch => {`

and then it should work